### ISSUE TITLE
Call back after video is played

### ISSUE THREAD

**Issue by saeedjassani** [PonnamKarthik/FlutterYoutube#3]
First of all, thanks for such a wonderful library. 

But there a few things which I think are essential.

There should be some way to receive callback once a video is done playing.

Also, there should be options to hide seekbar and other options.

**Comment by ponnamkarthik** [comment]
@saeedjassani Thanks your suggestion will add them in next release

**Comment by saeedjassani** [comment]
Hey @PonnamKarthik How is it going? Let me know if I can also contribute to solve the issue. Although I will need some help starting to understand the code

**Comment by ponnamkarthik** [comment]
@saeedjassani Check latest version 1.1.0 feature added for more details check readme

**Comment by saeedjassani** [comment]
Thanks @PonnamKarthik . Should I open new issue for iOS? I see it is still remaining in it.
